torch/onnx/onnx_supported_ops.py

```diff
@@ -0,0 +1,110 @@
+from inspect import signature, _empty  # type: ignore[attr-defined]
+from torch._C import _jit_get_all_schemas, FunctionSchema
+from torch.onnx.symbolic_registry import _registry, register_version
+from torch.onnx.symbolic_helper import _onnx_main_opset, _onnx_stable_opsets
+from typing import Dict, List, Union
+
+
+for v in _onnx_stable_opsets + [_onnx_main_opset]:
+    register_version("", v)
+
+class _TorchSchema:
+    def __init__(self, schema: Union[FunctionSchema, str]) -> None:
+        if isinstance(schema, FunctionSchema):
+            self.name: str = schema.name
+            self.overload_name: str = schema.overload_name
+            self.arguments: List[str] = [arg.name for arg in schema.arguments]
+            self.optional_arguments: List[str] = []
+            self.returns: List[str] = [ret.name for ret in schema.returns]
+            self.opsets: List[int] = []
+        else:
+            self.name = schema
+            self.overload_name = ""
+            self.arguments = []
+            self.optional_arguments = []
+            self.returns = []
+            self.opsets = []
+
+    def __str__(self) -> str:
+        s = f"{self.name}.{self.overload_name}("
+        s += ", ".join(self.arguments)
+        s += ") -> ("
+        s += ", ".join(self.returns)
+        s += ")"
+        s += " in opsets "
+        s += ", ".join(str(opset) for opset in self.opsets)
+        return s
+
+    def __hash__(self):
+        # TODO: handle overload_name?
+        return hash((self.name))
+
+    def __eq__(self, other) -> bool:
+        if not isinstance(other, _TorchSchema):
+            return False
+        # TODO: handle overload_name?
+        return self.name == other.name
+
+    def is_aten(self) -> bool:
+        return self.name.startswith("aten::")
+
+    def is_backward(self) -> bool:
+        return "backward" in self.name
+
+
+def _all_aten_forward_schemas():
+    """ Creates a list of _TorchSchema for all aten schemas"""
+    torch_schemas = [_TorchSchema(s) for s in _jit_get_all_schemas()]
+    torch_schemas = sorted(torch_schemas, key=lambda x: x.name)
+    aten_schemas = [s for s in torch_schemas if s.is_aten() and not s.is_backward()]
+    return aten_schemas
+
+
+def _symbolic_argument_count(func):
+    params = []
+    sig = signature(func)
+    optional_params = []
+    has_var = False
+    for name, p in sig.parameters.items():
+        if p.kind.name == "VAR_POSITIONAL":
+            has_var = True
```

Comment:
`has_var` never used?

Reply:
@justinchuby can you send a PR for this?

@kit1980 the original version was going to support aten variants, but we post ponded. This is certainly a left over. Thanks for pointing out!